docs/docs/how-tos/nebari-upgrade.md

```diff
@@ -4,15 +4,18 @@ title: Upgrade Nebari
 description: An overview of how to upgrade Nebari. Includes instructions on updating when breaking changes are expected to occur.
 ---
 
-# Upgrade Nebari from pre-0.4.0 to 0.4.0
+# Upgrading Nebari
```

Comment:
```suggestion
# Upgrade Nebari
```

Tiny sugegstion: We've decided to use [simple present tense across the docs](https://deploy-preview-286--nebari-docs.netlify.app/docs/community/style-guide#quick-reference)